template.distillery/PROJECT_NAME.sql

```diff
@@ -12,12 +12,14 @@ CREATE TABLE users ( -- DEFAULT
 
 CREATE TABLE emails ( -- DEFAULT
        email text primary key, -- DEFAULT
-       userid bigint NOT NULL references users(userid) -- DEFAULT
+       userid bigint NOT NULL references users(userid), -- DEFAULT
+       is_primary boolean NOT NULL,
+       is_activated boolean NOT NULL
 );
```

Comment:
Good catch !
Regarding the last question:
"Or maybe just activated email addresses should be unique, ..."
I guess you meant
"Or maybe just *primary* email addresses should be unique, ..." didn't you ?

The use case of a parent, setting the same recovery address to all of its children account*, might indeed be worth considering. I'd vote for 
"just *primary* email addresses should be unique".
Do you agree @vouillon ?

*: so that the child can still reset the password and use its account even if he/she forgot his/her password to the application and mail.